docs/servo_power_test/src/config.ts

```diff
@@ -4,17 +4,29 @@ import {ServoController} from './servo_controller';
 import {TestRunner} from './test_runner';
 import {Ui} from './ui';
 
+export class IterationData {
+  public powerDataList: Array<PowerData>;
+  public annotationList: AnnotationDataList;
+  constructor(
+    powerDataList: Array<PowerData>,
+    annotationList: AnnotationDataList
+  ) {
+    this.powerDataList = powerDataList;
+    this.annotationList = annotationList;
+  }
+}
+
 export class Config {
   private INTERVAL_MS = 100;
   private ui: Ui;
   private servoController: ServoController;
   private runner: TestRunner;
   private halt = true;
   private inProgress = false;
-  public powerDataList: Array<PowerData> = [];
-  public annotationList: AnnotationDataList = new Map<string, number>();
+  public iterationDataList: Array<IterationData> = [];
```

Comment:
Can you add methods to access iterationDataList to make it private? Less public fields is a good practice, if there are no strong reasons.
c.f. https://google.github.io/styleguide/tsguide.html#visibility